staging/src/k8s.io/apiserver/pkg/apis/audit/v1beta1/zz_generated.conversion.go

```diff
@@ -0,0 +1,272 @@
+limitations under the License.
+*/
+
+// This file was autogenerated by conversion-gen. Do not edit it manually!
+
+package v1beta1
+
+import (
+	authentication_v1 "k8s.io/api/authentication/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	conversion "k8s.io/apimachinery/pkg/conversion"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	types "k8s.io/apimachinery/pkg/types"
+	audit "k8s.io/apiserver/pkg/apis/audit"
+	unsafe "unsafe"
+)
+
+func init() {
+	localSchemeBuilder.Register(RegisterConversions)
+}
+
+// RegisterConversions adds conversion functions to the given scheme.
+// Public to allow building arbitrary schemes.
+func RegisterConversions(scheme *runtime.Scheme) error {
+	return scheme.AddGeneratedConversionFuncs(
+		Convert_v1beta1_Event_To_audit_Event,
+		Convert_audit_Event_To_v1beta1_Event,
+		Convert_v1beta1_EventList_To_audit_EventList,
+		Convert_audit_EventList_To_v1beta1_EventList,
+		Convert_v1beta1_GroupResources_To_audit_GroupResources,
+		Convert_audit_GroupResources_To_v1beta1_GroupResources,
+		Convert_v1beta1_ObjectReference_To_audit_ObjectReference,
+		Convert_audit_ObjectReference_To_v1beta1_ObjectReference,
+		Convert_v1beta1_Policy_To_audit_Policy,
+		Convert_audit_Policy_To_v1beta1_Policy,
+		Convert_v1beta1_PolicyList_To_audit_PolicyList,
+		Convert_audit_PolicyList_To_v1beta1_PolicyList,
+		Convert_v1beta1_PolicyRule_To_audit_PolicyRule,
+		Convert_audit_PolicyRule_To_v1beta1_PolicyRule,
+	)
+}
+
+func autoConvert_v1beta1_Event_To_audit_Event(in *Event, out *audit.Event, s conversion.Scope) error {
+	out.ObjectMeta = in.ObjectMeta
+	out.Level = audit.Level(in.Level)
+	out.Timestamp = in.Timestamp
+	out.AuditID = types.UID(in.AuditID)
+	out.Stage = audit.Stage(in.Stage)
+	out.RequestURI = in.RequestURI
+	out.Verb = in.Verb
+	// TODO: Inefficient conversion - can we improve it?
+	if err := s.Convert(&in.User, &out.User, 0); err != nil {
+		return err
+	}
+	out.ImpersonatedUser = (*audit.UserInfo)(unsafe.Pointer(in.ImpersonatedUser))
+	out.SourceIPs = *(*[]string)(unsafe.Pointer(&in.SourceIPs))
+	out.ObjectRef = (*audit.ObjectReference)(unsafe.Pointer(in.ObjectRef))
+	out.ResponseStatus = (*v1.Status)(unsafe.Pointer(in.ResponseStatus))
+	out.RequestObject = (*runtime.Unknown)(unsafe.Pointer(in.RequestObject))
+	out.ResponseObject = (*runtime.Unknown)(unsafe.Pointer(in.ResponseObject))
+	return nil
+}
+
+// Convert_v1beta1_Event_To_audit_Event is an autogenerated conversion function.
+func Convert_v1beta1_Event_To_audit_Event(in *Event, out *audit.Event, s conversion.Scope) error {
+	return autoConvert_v1beta1_Event_To_audit_Event(in, out, s)
+}
+
+func autoConvert_audit_Event_To_v1beta1_Event(in *audit.Event, out *Event, s conversion.Scope) error {
+	out.ObjectMeta = in.ObjectMeta
+	out.Level = Level(in.Level)
+	out.Timestamp = in.Timestamp
+	out.AuditID = types.UID(in.AuditID)
+	out.Stage = Stage(in.Stage)
+	out.RequestURI = in.RequestURI
+	out.Verb = in.Verb
+	// TODO: Inefficient conversion - can we improve it?
```

Comment:
The same happens in the v1alpha1 conversions as well. Not nice, not wrong, but suspicious. Maybe a bug in our conversion-gen.